server/src/interfaces/notification.interface.ts

```diff
@@ -1,3 +1,5 @@
+import { Attachment } from 'nodemailer/lib/mailer';
```

Comment:
there are no imports from `nodemailer` outside of the notification repository. I would suggest defining a similar interface here instead.